td/tests/publishing/test_forms.py

```diff
@@ -59,6 +59,12 @@ def test_rejected_fields(self):
         form = PublishRequestNoAuthForm({}, None)
         self.assertFalse('rejected_at' in form.fields)
 
+    def test_license_links(self):
+        form = PublishRequestNoAuthForm({}, None)
+        self.assertTrue('license_agreements' in form.fields)
+        self.assertTrue('https://unfoldingword.org/license' in
```

Comment:
Do we need to also check for the existence of `statement of faith` and `translation guidelines` being in form.fields.help_text?